### ISSUE TITLE
boinc not downloading tasks in openSUSE 12.1-12.2 32-bit

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#1200]
**Reported by Tzeentch on 27 Feb 43192199 06:21 UTC**
when using boinc either as a daemon with systemd or through the GUI with the manager boinc doesn't download tasks. the service will run without an issue. in the gui it reports not enough disk space as the cause even if you assign 100% in the prefrences locally and through the projects settings it reports you need more disk space you have 0.00MB free.

On the headless server it reports master_fetch failures when contacting the projects. I tried different projects. I was able to replicate this on two different machines one with suse 12.1 the other 12.2 both 32 bit. 

Migrated-From: http://boinc.berkeley.edu/trac/ticket/1228

**Comment by romw** [comment]
**Commented by Tzeentch on 30 Aug 43192202 22:15 UTC**
Replying to [Tzeentch](ticket:1228):
> when using boinc either as a daemon with systemd or through the GUI with the manager boinc doesn't download tasks. the service will run without an issue. in the gui it reports not enough disk space as the cause even if you assign 100% in the prefrences locally and through the projects settings it reports you need more disk space you have 0.00MB free.
> 
> On the headless server it reports master_fetch failures when contacting the projects. I tried different projects. I was able to replicate this on two different machines one with suse 12.1 the other 12.2 both 32 bit. 

I have ensured my authentication settings when headless are correct as it will attach to the project. Also chmod + chown all files in the /var/lib/boinc dir's for boinc too when it was a daemon

**Comment by romw** [comment]
**Commented by Tzeentch on 11 May 43192208 05:28 UTC**
forgot to add in that the boinc version is from downloading from yast repos. didn't compile or use berkely installer

**Comment by romw** [comment]
**Commented by ChristianB on 19 Oct 43193408 07:14 UTC**
1. this is a somewhat ancient version of BOINC which is not updated anymore
2. can you post the messages about the disk space issues and master file fetch errors you get?
3. Normally I would advise you do turn on some debugging flags but I don't know if they are present in 6.10.60 (it's work_fetch_debug in [cc_config.xml](http://boinc.berkeley.edu/wiki/Cc_config.xml#Logging_flags))
4. If you feel adventurous you can try to install the latest berkeley boinc into /var/lib/boinc and try again

**Comment by romw** [comment]
**Commented by Tzeentch on 14 Feb 43193869 13:32 UTC**
i'd use the berkeley installer but it installs BOINC in a way that fails to take advantage of Linux security features and BOINC binaries use more memory than they really need to because several shared libraries are statically linked. 

i'd prefer the package management method. so if this version is very old how can I get the repositories to use a more up to date version of the software? as anyone using suse using the pack management would likely face the same issues

**Comment by romw** [comment]
**Commented by ChristianB on 18 Jan 43193902 20:27 UTC**
There is a more up to date (latest 6.x release) package for openSuSe 12.2 available (http://software.opensuse.org/package/boinc-client or http://pkgs.org/download/boinc-client) I don't know if you can use this on 12.1 also. I'm not familiar with the !SuSe policy but for Debian or Ubuntu you would contact the package maintainer and ask for an upgrade. Browsing around some of the pages I mentioned I found [this](https://build.opensuse.org/package/show?package=boinc-client&project=openSUSE%3A12.2) where you can report bugs and requests but no info how to contact the maintainer. There is also an unstable version of 7.0.54 in what looks like a personal repository.

Sorry, I can't help you there, I'm a Debian user myself and only tried !SuSe only once, years ago.

**Comment by ChristianBeer** [comment]
Closing as EOL. Please open a new issue if this still happens on more recent SuSe versions.
